en/ros/offboard_control.md

```diff
@@ -1,6 +1,6 @@
-# Offboard Control 
```

Comment:
FYI this doc is first doc in the  Robotics API section. It is supposed to explain how offboard control works/what it is.

The concept is useful but this is so out of date it probably should be deleted. This is added to make sure I look at this topic next week and decide what can be used or to discard it.